charts/gardener-extension-shoot-cert-service/values.yaml

```diff
@@ -27,6 +27,7 @@ controllers:
     concurrentSyncs: 5
 
 certificateConfig:
+  defaultRequestsPerDayQuota: 100
```

Comment:
For backwards compatibility can you remove or comment this line?

Reply:
done